testing/src/components/Container.js

```diff
@@ -30,24 +75,30 @@ export const Container = () => {
       <div className="app-title">the TODO LIST</div>
       <div className="form">
         <div className="input-title">Add a Todo Item</div>
-        <Input setTodoName={setTodoName} todoName={todoName} />
-        <button className="Add-Item-Button" onClick={handleClick}>
-          Add item
-        </button>
-
-        <div className="" style={{ marginTop: "4vmin" }}>
-          <div style={{ paddingBottom: "4vmin" }}>TODOs:</div>
-          {itemsToDo.map((item) => (
-            <div
-              style={{
-                backgroundColor: "white",
-                padding: "4vmin",
-                borderRadius: "2vmin",
-              }}
-            >
-              {item}
-            </div>
-          ))}
+        <div>
+          <Input setTodoName={setTodoName} todoName={todoName} />
+          <button className="Add-Item-Button" onClick={handleClick}>
+            Add item
+          </button>
+
+          {items.map((item, index) => {
+            return (item.edit && (isEdit === 1)) ? (
+              <ItemEdit
+                updateVal={updateVal}
+                onSaveEdit={onSaveEdit}
+                cancelEditItem={cancelEditItem}
+                item={item}
+                index={index}
```

Comment:
you wanna do `key={index}` in this situation to stop the warning

Reply:
gotcha!